modules/shared/php/options-utilities.php

```diff
@@ -8,18 +8,45 @@
 namespace VIPWorkflow\Modules\Shared\PHP;
 
 use stdClass;
-use VIPWorkflow\VIP_Workflow;
 
 class OptionsUtilities {
+	const OPTIONS_GROUP = 'vip_workflow_';
+	const OPTIONS_GROUP_NAME = 'vip_workflow_options';
+
+	// Its key to centralize these here, so we can easily update them in the future
+	private const DEFAULT_OPTIONS = [
```

Comment:
I've kept the default options here but I'd like to move this eventually to a typed class that we can rely on easily.